### PR TITLE
Add support for RML language

### PR DESCRIPTION
**Pull request by pie-catt**
Add support for RML language.
https://rmlatdibris.github.io/

RML is a rewriting-based and system agnostic Domain Specific Language for Runtime Verification which decouples monitoring from instrumentation by allowing users to write specifications and to synthesize monitors from them, independently of the System Under Scrutiny (SUS) and its instrumentation.

RML has been developed and is currently maintained by [DIBRIS](https://www.dibris.unige.it/en/) - University of Genoa

RML_sample

### CODE REVIEW

**Comment by pie-catt** [comment]
Hello @tancnle. Are there any updates on this? Thank you for your work

**Comment by tancnle** [comment]
Thanks for the ping @pie-catt. I will have a look at it this week. Thank you for your patience. Really appreciated 🙇🏼

File: lib/rouge/lexers/rml.rb [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+    module Lexers
+    class RML < RegexLexer
+      title "RML"
+      desc "A system agnostic domain specific language for runtime monitoring and verification"
```
Comment (tancnle) [inline lib/rouge/lexers/rml.rb:8]: **suggestion:** Could we add a URL to the language spec page? WDYT?

```suggestion
      desc "A system agnostic domain-specific language for runtime monitoring and verification (https://rmlatdibris.github.io/)"
```

File: lib/rouge/lexers/rml.rb [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+    module Lexers
```
Comment (tancnle) [inline lib/rouge/lexers/rml.rb:5]: **suggestion:** We use 2-space indentation in our code base. Would it be possible to fix all the indentations on this file? 🙏🏼

**Review by tancnle — COMMENTED** [review]
Thank you for your contribution @pie-catt ❤️ I have added some suggestions for your consideration. Please let me know what you think.

**Comment by pie-catt** [comment]
Thank you so much for your support @tancnle 🙏🏼. I updated my pull request with your suggestions. I fixed code indentation, renamed variables names to snake-case, removed superfluous and closing comments. I also added RML language entry in [Languages.md](https://github.com/rouge-ruby/rouge/blob/master/docs/Languages.md) file. Thank you for your work

**Comment by tancnle** [comment]
Thank you for considering the suggestions @pie-catt 🙇🏼 LGTM. Let's launch this into orbit 🚀

**Pull request merged**